packages/app-headless-cms/src/admin/views/contentEntries/ContentEntry/FullScreenContentEntry/FullScreenContentEntryHeaderLeft.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from "react";
+import { ReactComponent as BackIcon } from "@material-design-icons/svg/round/arrow_back.svg";
+import { useNavigateFolder } from "@webiny/app-aco";
+import { IconButton } from "@webiny/ui/Button";
+import { Typography } from "@webiny/ui/Typography";
+import { useModel } from "~/admin/components/ModelProvider";
+import { useContentEntry } from "~/admin/views/contentEntries/hooks";
+import {
+    EntryMeta,
+    EntryTitle,
+    EntryVersion,
+    TitleWrapper,
+    EntryName
+} from "./FullScreenContentEntry.styled";
+
+export const FullScreenContentEntryHeaderLeft = () => {
+    const { model } = useModel();
+    const { entry } = useContentEntry();
+    const { navigateToFolder, currentFolderId } = useNavigateFolder();
+
+    const title = useMemo(
+        () => entry?.meta?.title || `New ${model.name}`,
+        [entry.meta, model.name]
+    );
```

Comment:
These type of memoizations are adding more calculations than they're optimizing. These simple assingments can be executed on each render, especially since the output is a primitive `string`.